### ISSUE TITLE
 Anvil Pets cannot repair items in equipment

### ISSUE THREAD

**Issue by CrayonYeser** [capnkirok/Inventory-Pets#626]
version 1.19.2 Forge 43.4.0
 Anvil Pets can repair items in bag but cannot repair items in equipment

**Comment by capnkirok** [comment]
Thanks. I'll check it out. I just made a fix, so maybe I did something wrong on the 1.19.2 version.

**Comment by capnkirok** [comment]
Indeed, I copied some code from 1.20 version that didn't translate. Fixed in the next release, version 2.2, look for it in about a week. Thanks for your patience!

**Comment by capnkirok** [comment]
Also affected the living entity... and on 1.20 version as well. 

Bad code 🤦
